### ISSUE TITLE
stat : no such file or directory

### ISSUE THREAD

**Issue by domdip** [PowerDataHub/terraform-aws-airflow#10]
apply failed with:
```
Error: Error applying plan:

3 error(s) occurred:

* module.airflow-cluster.aws_instance.airflow_webserver: stat : no such file or directory
* module.airflow-cluster.aws_instance.airflow_worker: stat : no such file or directory
* module.airflow-cluster.aws_instance.airflow_scheduler: stat : no such file or directory
```

Config was as in this issue: 
https://github.com/PowerDataHub/terraform-aws-airflow/issues/9

I think the issue may be that the package does not contain requirements.txt and I did not specify a custom one.  (I see .txt in Github but it may not have been included in the module.  Not seeing it in .terraform)

Terraform has an [open issue](https://github.com/hashicorp/terraform/issues/15932) where missing files will cause the file provisioner to fail with a cryptic message.

~~Can work around it by creating an empty requirements.txt.~~

Edit:  still encountering this after providing requirements.txt.  Unfortunately terraform doesn't say which resource is missing.

**Comment by edbizarro** [comment]
Weird because if none of the files are present the module use the placeholders https://github.com/PowerDataHub/terraform-aws-airflow/blob/master/variables.tf#L241-L247

This issue starts after the "agent" change?

**Comment by domdip** [comment]
Hmm actually I cloned the repo and imported the module locally (where I can clearly see the files it might refer to).  Same error.

This happens with the current master (agent = false).  Haven't tried with agent, will give that a go shortly.

This is an annoying place to fail because create_before_destroy never gets around to destroying the instance.  Try enough times and you hit an instance limit.

**Comment by edbizarro** [comment]
I can confirm this, I was able to reproduce in my local environment.

**Comment by edbizarro** [comment]
Fixed on 9d76eb751881397686c5470f1dbcf454f8c0bb32

Looks like I was using the wrong approach (conditionals) when all I needed was a coalesce function

**Comment by domdip** [comment]
Confirmed this is working after update, thanks!
